packages/lab/src/tabs-next/TabNext.css

```diff
@@ -0,0 +1,150 @@
+/* Class applied to root Tab element */
+.saltTabNext {
+  --saltEditableLabel-padding: 0;
+  --saltEditableLabel-height: var(--tabs-tabstrip-height);
+  --saltInput-minWidth: 4em;
+
+  --tabs-tab-background: var(--salt-navigable-primary-background);
+  --tabs-tab-background-hover: rgba(0, 0, 0, 0.08);
```

Comment:
whats this color?

Reply:
This is the new opacity that doesn't exist as a characteristic yet (0.08). I suggested we hard-code this as it is a potential one off, and also we cannot use opacities in rgba() constructor anyways.